internal/cmd/server.go

```diff
@@ -34,8 +35,9 @@ func serverCmd() *cobra.Command {
 	cmdconfig.
 		OnCmd(cmd).
 		AddIntFlag(constants.ArgPort, localconstants.DefaultServerPort, "Server port.").
-		AddStringFlag(constants.ArgListen, localconstants.DefaultListen, "listen address port.").
-		AddBoolFlag(constants.ArgWatch, true, "Watch mod files for changes when running Flowpipe server")
+		AddStringFlag(constants.ArgListen, localconstants.DefaultListen, "Listen address port.").
+		AddBoolFlag(constants.ArgWatch, true, "Watch mod files for changes when running Flowpipe server").
```

Comment:
Should have full stop at the end of the command description just like the others.